custom/covid/management/commands/clear_owner_ids.py

```diff
@@ -0,0 +1,45 @@
+from xml.etree import cElementTree as ElementTree
+
+from casexml.apps.case.mock import CaseBlock
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from custom.covid.management.commands.update_cases import CaseUpdateCommand
+
+BATCH_SIZE = 100
+DEVICE_ID = __name__ + ".update_owner_ids"
+
+
+class Command(CaseUpdateCommand):
+    help = "Makes the owner_id for cases blank"
+
+    def case_block(self, case):
+        blank_owner_id = {'owner_id': '-'}
+        return ElementTree.tostring(CaseBlock.deprecated_init(
```

Comment:
```suggestion
        return ElementTree.tostring(CaseBlock(
```

For context:
https://github.com/dimagi/commcare-hq/blob/f0e5d829b75315188ce707ad299b16f0f0b32fc9/corehq/ex-submodules/casexml/apps/case/mock/case_block.py#L61-L69

Reply:
Nice, added.